rflink/__main__.py

```diff
@@ -22,14 +22,17 @@
 import asyncio
 import logging
 import sys
+from typing import Dict, Optional, Sequence, Type  # noqa: unused-import
 
 import pkg_resources
 from docopt import docopt
 
-from .protocol import (
+from .protocol import (  # noqa: unused-import
```

Comment:
Unused imports should be removed, if you want to import something to have it available in this module for import by another module include it in `__all__`.

Reply:
Those imports are needed by the `# type: Dict[str, Type[ProtocolBase]]` some lines below -- without them mypy will yell, and with them pyflakes will yell without the `noqa`.